libs/markets/src/lib/markets-provider.ts

```diff
@@ -154,13 +154,13 @@ export const marketListProvider = makeDerivedDataProvider<
 );
 
 export const useMarketList = () => {
-  const yesterday = useYesterday();
+  const fiveDaysAgo = useFiveDaysAgo();
```

Comment:
when a network upgrade happens, we might not get the last 24h candles accurately. 
it'd be good to fetch last 5 days candles and then filter which candles are from 24h period.